pkg/minikube/console/style.go

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright 2019 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package console
+
+import (
+	"fmt"
+
+	"golang.org/x/text/message"
+)
+
+// style describes how to stylize a message.
+type style struct {
+	// Prefix is a string to place in the beginning of a message
+	Prefix string
+	// OmitNewline omits a newline at the end of a message.
+	OmitNewline bool
+}
+
+// styles is a map of style name to style struct
+// For consistency, ensure that emojis added render with the same width across platforms.
+var styles = map[string]style{
+	"happy":      {Prefix: "😄"},
+	"success":    {Prefix: "✅"},
+	"failure":    {Prefix: "❌"},
+	"conflict":   {Prefix: "💥"},
+	"fatal":      {Prefix: "💣"},
+	"notice":     {Prefix: "📌"},
+	"ready":      {Prefix: "🏄"},
+	"restarting": {Prefix: "🔄"},
+	"stopping":   {Prefix: "✋"},
+	"stopped":    {Prefix: "🛑"},
+	"warning":    {Prefix: "⚠️"},
+	"waiting":    {Prefix: "⌛"},
+	"usage":      {Prefix: "💡"},
+	"launch":     {Prefix: "🚀"},
+	"thumbs-up":  {Prefix: "👍"},
+	"option":     {Prefix: "   ▪ "},
```

Comment:
This one has an extra space at the end - is that intentional? 
```suggestion
	"option":     {Prefix: "   ▪"},
```

Reply:
Thanks for pointing this out. I've switched it so that styles now define their own trailing whitespace, as this one is slightly different.